src/who.md

```diff
@@ -79,7 +79,7 @@ order):
   jitendra         |[Jitendra Nath Pandey](http://people.apache.org/~jitendra)                   |Hortonworks    |            |-8
   jlowe            |Jason Lowe                                                                   |NVIDIA         |            |-6
   jzhuge           |John Zhuge                                                                   |Netflix        |            |-8
-  junping\_du      |Junping Du                                                                   |Datastrato     |            |-8
+  junping\_du      |Junping Du                                                                   |Datastrato     |            |+8
```

Comment:
Would you update the who.md in a separate PR?

Reply:
If need I can try to separate them. At this PR, I try to submit different commits for the three changes at one PR as description says.

Reply:
ref: https://github.com/apache/hadoop-site/pull/49/commits